portal-api/api-tests/v1/deals/mocks.js

```diff
@@ -30,6 +30,7 @@ const updatedECCriteria11NoExtraInfo = {
   'criterion-11': 'false',
   'criterion-12': 'true',
   'criterion-14': 'false',
+  'agentAddressCountry': 'GBR',
```

Comment:
Surprising!

Reply:
I cant comment on lines that arent editted, but if this is needed here, we can remove it from line 39